sdk/src/main/java/com/ifountain/opsgenie/client/model/beans/TeamLog.java

```diff
@@ -7,7 +7,7 @@
 public class TeamLog extends Bean {
     private String log;
     private String owner;
-    private long createdAt;
+    private String createdDate;
```

Comment:
Should it be a Date object?

Reply:
As per document -https://docs.opsgenie.com/docs/team-api#list-team-logs,
Response for list team logs:
`{
    "data": {
        "offset": "1495523940765000268",
        "logs": [
            {
                "owner": "System",
                "createdDate": "2017-05-23T07:45:51.56Z",
                "log": "Updated members."
            },
            {
                "owner": "System",
                "createdDate": "2017-05-23T07:19:00.765Z",
                "log": "Created team \"TeamName\"."
            }
        ]
    },
    "took": 0.041,
    "requestId": "1cee4c3e-a573-48b0-bef5-alpbfb03da63"
}`

Here createdDate is a string, so that's why have to change its data type. For all the dates which are taken string as input, we have put a validation check for that.

Reply:
You can use date object here instead of a string: for example:
@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss'Z'")
private Date currentTime;